packages/api/src/external/fhir/document/index.ts

```diff
@@ -109,6 +108,33 @@ export const toFHIR = (
   };
 };
 
+export function createMetriportDocReference({
```

Comment:
Great that we have this as a function now!

nit: the name of the function conveys the idea that it creates/returns a `DocumentReference`. Maybe add `Content` as a suffix?